cfme/base/ui.py

```diff
@@ -62,3 +107,612 @@ class Dashboard(CFMENavigateStep):
+        self.view.accordions.diagnostics.tree.click_path(
+            self.obj.zone.region.settings_string,
+            "Zone: {} (current)".format(self.obj.zone.description),
+            "Server: {} [{}] (current)".format(
+                self.obj.name, self.obj.sid))
+
+
+@navigator.register(Server)
+class DiagnosticsDetails(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.summary.select()
+
+
+@navigator.register(Server)
+class DiagnosticsWorkers(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.workers.select()
+
+
+class DiagnosticsCollectLogsView(ServerDiagnosticsView):
+    edit = Button(title="Edit the Log Depot settings for the selected Server")
+
+
+@navigator.register(Server)
+class DiagnosticsCollectLogs(CFMENavigateStep):
+    VIEW = DiagnosticsCollectLogsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.collectlogs.select()
+
+
+@navigator.register(Server)
+class DiagnosticsLogsSettings(CFMENavigateStep):
+    VIEW = DiagnosticsCollectLogsView
+    prerequisite = NavigateToSibling('DiagnosticsCollectLogs')
+
+    def step(self):
+        self.view.edit.click()
+
+
+@navigator.register(Server)
+class Log(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.cfmelog.select()
+
+
+@navigator.register(Server)
+class AuditLog(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.auditlog.select()
+
+
+@navigator.register(Server)
+class ProductionLog(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.productionlog.select()
+
+
+@navigator.register(Server)
+class Utilization(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.utilization.select()
+
+
+@navigator.register(Server)
+class Timelines(CFMENavigateStep):
+    VIEW = ServerDiagnosticsView
+    prerequisite = NavigateToSibling('Diagnostics')
+
+    def step(self):
+        self.view.timelines.select()
+
+
+# ######################## REGION NAVS ################################
+
+class RegionView(ConfigurationView):
+    @View.nested
+    class details(Tab):  # noqa
+        TAB_NAME = "Details"
+
+    @View.nested
+    class canducollection(Tab):  # noqa
+        TAB_NAME = "C & U Collection"
+
+    @View.nested
+    class redhatupdates(Tab):  # noqa
+        TAB_NAME = "Red Hat Updates"
+
+    @View.nested
+    class imports(Tab):  # noqa
+        TAB_NAME = "Import"
+
+    @View.nested
+    class importtags(Tab):  # noqa
+        TAB_NAME = "Import Tags"
+
```

Comment:
Yes, those are actually stored somewhere else as class methods, but I didn't turn them into collections yet, so, I'm not sure which is the best place to have them.